resources/services/kinesis/gen.hcl

```diff
@@ -45,9 +45,255 @@ resource "aws" "kinesis" "streams" {
       params = ["stream_arn"]
     }
   }
+  userDefinedColumn "tags" {
+    type              = "json"
+    generate_resolver = true
+  }
+}
 
+resource "aws" "kinesis" "firehoses" {
+  path = "github.com/aws/aws-sdk-go-v2/service/firehose/types.DeliveryStreamDescription"
+  ignoreError "IgnoreAccessDenied" {
```

Comment:
do we want different sets of ignore errors?
We have ignoreCommonErrors that encompasses most of the issues, why not just use it?

Reply:
Copy and paste mistake/oversight